scripts/ci

```diff
@@ -5,6 +5,9 @@ cd $(dirname $0)
 
 ./validate
 ./build
-./test
+# skipping tests for s390x since k3s doesn't support s390x yet
+if [ ${ARCH} != s390x ]; then
+    ./test
```

Comment:
i think i would prefer this being in test itself to return 0 at the top of the file if it's z. i don think the ci stub should care if it's on z or not

Reply:
Done.